tests/cypress/support/commands.ts

```diff
@@ -38,6 +38,6 @@ Cypress.Commands.add('restoreLocalStorage', () => {
 
 Cypress.Commands.add('safeClick', { prevSubject: 'element' }, ($element) => {
     const click = ($el) => $el.click()
-    return cy.wrap($element).should('exist').should('be.visible').pipe(click)
+    return cy.wrap($element).should('exist').should('be.visible').wait(500).pipe(click)
```

Comment:
This is a fine solution for now. We have tried several other things before getting to this point and it seems like the hit to our cypress tests time is acceptable to the team. Let's keep an eye on Cypress dashboard after this merges to see how much slower things get and watch out if we keep needing to bump this number higher and higher in the future to get same results.